res/contribute.json

```diff
@@ -10,23 +10,13 @@
         "home": "https://github.com/firefox-devtools/profiler/blob/main/CONTRIBUTING.md",
         "docs": "https://github.com/firefox-devtools/profiler/tree/main/docs-developer",
         "matrix": "https://chat.mozilla.org/#/room/#profiler:mozilla.org",
-        "slack": "https://devtools-html-slack.herokuapp.com/",
-        "irc": "irc://irc.mozilla.org/#devtools",
-        "irc-contacts": [
-          "gregtatum",
-          "julienw",
-          "florian",
-          "canova",
-          "gerald"
-        ],
         "chat": {
           "url": "https://chat.mozilla.org/#/room/#profiler:mozilla.org",
           "contacts": [
-            "@gregtatum:mozilla.com",
-            "@julienw:mozilla.com",
-            "@florian:mozilla.com",
-            "@canova:mozilla.com",
-            "@gerald:mozilla.com"
+            "@julienw:mozilla.org",
+            "@canova:mozilla.org",
+            "@mstange:mozilla.org",
+            "@davehunt:mozilla.org"
```

Comment:
Would it be good to add Dave as a team member above too?

Reply:
ah you're right, better be consistent